search/lib/cmr.js

```diff
@@ -29,41 +29,94 @@ const DEFAULT_HEADERS = {
 };
 
 /**
- * Convert a STAC Collection ID to set of CMR query parameters
- * @param {string} providerId The CMR Provider ID
- * @param {string} collectionId The STAC Collection ID
+ * Query a CMR Search endpoint with optional parameters
+ * @param {string} path CMR path to append to search URL (e.g., granules.json, collections.json)
+ * @param {object} params Set of CMR parameters
  */
-function stacCollectionToCmrParams (providerId, collectionId) {
-  const collectionIds = collectionId.split('.v');
-  const version = collectionIds.pop();
-  const shortName = collectionIds.join('.');
-  return {
-    provider_id: providerId,
-    short_name: shortName,
-    version
-  };
+async function cmrSearch (path, params) {
+  // should be search path (e.g., granules.json, collections, etc)
+  if (!path || !params) throw new Error('Missing url or parameters');
+  const url = makeCmrSearchUrl(path);
+  logger.debug(`CMR Search: ${url} with params: ${JSON.stringify(params)}`);
```

Comment:
is this needed?

Reply:
I think so, it's helpful to have these debug statement be in the code for local testing and deployment to UAT and SIT. Production has log level set to INFO so they aren't output.